cmd/goci/main.go

```diff
@@ -200,9 +200,8 @@ func validateRun() error {
 		return &ValidationError{Message: fmt.Sprintf("Your applications go version %v is no longer supported. Please upgrade to version %v.", repoVersion, newestGoVersion)}
 	} else if repoVersion <= newestGoVersion-0.01 {
 		// We'll give a PR comment to the Author to warn them about the need to upgrade
-		fmt.Printf("Warning: This applications go version will be out of support by the next major release. You will have until the next release before you need to upgrade to version %v\n", newestGoVersion)
+		fmt.Printf("A new Go version is out. Your current version (%v) is supported by the Go team for an additional 6 months. After that, CI will fail if it is not upgraded.", f.Go.Version)
```

Comment:
The comment doesn't tell me what date the support stops. If new version is out today, the comment will say "6 months" today and even month from now. 

It would be great to print something like "A new Go version was released on ..... After 6 months of release .... CI will fail" (my point is to add dates, not exact copy)

Reply:
https://go.dev/doc/devel/release has the release dates